en/docs/concepts/oidc/hybrid-client-profile.md

```diff
@@ -0,0 +1,242 @@
+in an authorization request selects the hybrid flow to be the
+authentication flow, let's take a look at the following
+` response_type ` values in detail .
+
+### code token
+
+This ` response_type ` requests a code and an access
+token from the authorization endpoint. 
+ 
+Following is a sample authorization request that uses
+` code token ` as the ` response_type ` :
+
+``` java
+https://localhost:9443/oauth2/authorize?response_type=code token&client_id=<Client ID>&nonce=asd&redirect_uri=http://localhost:8080/playground2/oauth2client&scope=openid
+```
+
+Following is the sample response that you will receive on successful
+authorization:
+
+``` java
+http://localhost:8080/playground2/oauth2client#access_token=1940a308-d492-3660-a9f8-46723cc582e9&code=99b34587-5483-374d-8b25-50485498e761&token_type=Bearer&expires_in=299999&session_state=baae9a71cdabe38b4643b9d59bd9f65ffaf5a9b8c453f4256c085e5a1c57e624.-EA3ZqPzLvsk25CKmt56YA
+```
+
+You can send the code to the token endpoint to request for an access
+token, refresh token and id\_token.
+<a name="curl"></a>
+Following is the curl command to request tokens from the token endpoint:
+
+``` java
+curl -k -v — user <Client ID>:<Client secret> -d “grant_type=authorization_code&code=99b34587–5483–374d-8b25–50485498e761&redirect_uri=http://localhost:8080/playground2/oauth2client" https://localhost:9443/oauth2/token
+```
+
+The response from the token endpoint will be as follows:
+
+``` java
+{“access_token”:”1940a308-d492–3660-a9f8–46723cc582e9",”refresh_token”:”6b96cc3a-00da-3d7d-acd1–5aaf76dcd9d4",”scope”:”openid”,”id_token”:”eyJ4NXQiOiJOVEF4Wm1NeE5ETXlaRGczTVRVMVpHTTBNekV6T0RKaFpXSTRORE5sWkRVMU9HRmtOakZpTVEiLCJraWQiOiJOVEF4Wm1NeE5ETXlaRGczTVRVMVpHTTBNekV6T0RKaFpXSTRORE5sWkRVMU9HRmtOakZpTVEiLCJhbGciOiJSUzI1NiJ9.eyJhdF9oYXNoIjoiSnJaWTlNdFlWRUlJSlV4LUREQm13dyIsInN1YiI6ImFkbWluIiwiYXVkIjpbIm5jemJnNW01eHh0NnRQNFVNWndCNlB0UW9Rb2EiXSwiYXpwIjoibmN6Ymc1bTV4eHQ2dFA0VU1ad0I2UHRRb1FvYSIsImlzcyI6Imh0dHBzOlwvXC9sb2NhbGhvc3Q6OTQ0M1wvb2F1dGgyXC90b2tlbiIsImV4cCI6MTUxMDgzMTAxMCwibm9uY2UiOiJhc2QiLCJpYXQiOjE1MTA4MzEwMDd9.XKV0ioEvflR4MHGthO3cwXwC88msNgqR4l1O83mfhxOMtO1PG3ABWB5E4aFXFpR9t-8zJs09slhLsDTDhmC33KE8Die61UK9_Vb5aNA4XCkawyJt8dCX6clc6UUbTEO5N1ubXA18QFgwAEWpvoTz1hKx8XLnvOSehbdEKsoPunoHDmXpYJe_9hBg5V3kN-VHxdKdGOtl9u-Aml42s5p45cZY0mlFVcKjatBAf7hqWNPlUebyujDWG1Iyk_-AXNQ2wYi0F77uG7_HstP_tp0sOctu0TYCK8bwBTXEJYMPt1CqOqcae05m8N8hb0zs6Yxvyx_udCJPG-8n2zRB-T-kcg”,”token_type”:”Bearer”,”expires_in”:299494}
+```
+
+When you decrypt the ` id_token ` it will be as follows:
+
+``` java
+{
+ “at_hash”: “JrZY9MtYVEIIJUx-DDBmww”,
+ “sub”: “admin”,
+ “aud”: [
+ “nczbg5m5xxt6tP4UMZwB6PtQoQoa”
+ ],
+ “azp”: “nczbg5m5xxt6tP4UMZwB6PtQoQoa”,
+ “iss”: “https://localhost:9443/oauth2/token",
+ “exp”: 1510831010,
+ “nonce”: “asd”,
+ “iat”: 1510831007
+}
+```
+
+There can be instances where you get two access tokens. One from the
+authorization endpoint and the other from the token endpoint. These two
+access tokens may or may not be same.
```

Comment:
There may be instances where two access tokens are received; one from the authorization endpoint, and the other from the token endpoint. These two access token may or may not be the same.